controllers/replication-controller/persistentvolume_controller.go

```diff
@@ -350,6 +350,33 @@ func (r *PersistentVolumeReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			return ctrl.Result{}, err
 		}
 
+		// Watson: If we're creating the remote PV, let's add a claimRef so the PV can only be used with
+		// a pvc of the specified namespace/name. Criteria
+		// 1. Should have no remoteVolume annotation, this is only on source and is the volume handle for the remove volume
+		// 2. The remotePVCNamespace and remotePVName must be supplied.
+		if createRemotePV {
```

Comment:
There are 2 blocks with the same 'createRemotePV' condition.. can we make these a single block ?

Reply:
Combined both the blocks